src/methods.ts

```diff
@@ -73,8 +73,6 @@ export const logReq = async (key: string, app: string) => {
 
     if (!keyData?.key) return { error: 'Key does not exist', code: 401 };
     if (!keyData?.active) return { error: 'Key is not active', code: 401 };
-    if (keyData?.month_total >= limits[app].monthly)
-      return { error: 'Key is restricted for this month', code: 429 };
```

Comment:
Even if the count exceeds, we keep counting, but the API key will work with default rate limit